packages/components/pages/f-mfa/src/components/Help.vue

```diff
@@ -29,7 +29,10 @@
                     <i18n
                         path="helpInfo.instructionsPoint4Text"
                         tag="li">
-                        <a :href="$t('helpInfo.instructionsHelpLink')"><strong>{{ $t('helpInfo.instructionsHelpText') }}</strong></a>
+                        <a
```

Comment:
❓- Will the instructions help link be another page in coreweb? If so could consider a `nuxt-link`

Reply:
No, I think it goes to consumer help

Reply:
Nope, it'll be linking to a help centre article